app/src/main/java/de/kai_morich/simple_bluetooth_le_terminal/SerialSocket.java

```diff
@@ -32,6 +32,8 @@
     private static final UUID BLUETOOTH_LE_NRF_SERVICE    = UUID.fromString("6e400001-b5a3-f393-e0a9-e50e24dcca9e");
     private static final UUID BLUETOOTH_LE_NRF_CHAR_RW2   = UUID.fromString("6e400002-b5a3-f393-e0a9-e50e24dcca9e"); // read on microbit, write on adafruit
     private static final UUID BLUETOOTH_LE_NRF_CHAR_RW3   = UUID.fromString("6e400003-b5a3-f393-e0a9-e50e24dcca9e");
+    private static final UUID BLUETOOTH_LE_RN4870_SERVICE = UUID.fromString("49535343-FE7D-4AE5-8FA9-9FAFD205E455");
+    private static final UUID BLUETOOTH_LE_RN4870_RW      = UUID.fromString("49535343-1E4D-4BD9-BA61-23C647249616");

```

Comment:
for consistency please change name to ..._CHAR_RW

Reply:
Ok, done